tools/devctr/Dockerfile.x86_64

```
@@ -74,9 +75,10 @@ RUN mkdir "$TMP_BUILD_DIR" \
         && rustup target add x86_64-unknown-linux-musl \
         && rustup component add rustfmt \
         && rustup component add clippy-preview \
+        && rustup install "$RUST_TOOLCHAIN_AUDIT" \
         && cd "$TMP_BUILD_DIR" \
             && cargo install cargo-kcov \
-            && cargo install cargo-audit \
+            && cargo +"$RUST_TOOLCHAIN_AUDIT" install cargo-audit \
```

Should we maybe use the latest toolchain available for cargo-audit, WDYT?

---

Updated to use v1.47.0.

---

nit: I was thinking we could do:

```suggestion
            && cargo +stable install cargo-audit \
```

and ideally we could also do `rustup update` to get latest toolchain

---

Yup, this is what I was referring to when i suggested using the latest available toolchain.